addons/web/static/src/core/dialog/dialog.js

```diff
@@ -39,6 +39,17 @@ export class Dialog extends Component {
         useHotkey("escape", () => {
             this.data.close();
         });
+        useHotkey(
+            "control+enter",
+            () => {
+                document
+                    .querySelector(
+                        ".o_dialog:not(.o_inactive_modal) .modal-footer button:not(.d-none)"
+                    )
+                    .click();
+            },
+            { bypassEditableProtection: true }
```

Comment:
maybe use the option isAvailable to check that a button is there --> I think there is sometimes no button